crates/subspace-service/src/sync_from_dsn/snap_sync.rs

```diff
@@ -110,14 +110,9 @@ where
                 debug!("Snap sync finished successfully");
             }
             Err(error) => {
-                // A fatal snap sync error requiring user intervention. The caller will log this error,
-                // so just return it to terminate the task.
-                if matches!(error, Error::SnapSyncImpossible(_)) {
-                    return Err(error);
-                }
+                error!(%error, "Snap sync failed: wipe the DB folder and rerun with --sync=full");
```

Comment:
This is only necessary in case of impossible error. There is nothing catastrophic otherwise and simple restart will do.

Reply:
what if I add " if the problem persists"?

Reply:
I don't understand how wiping database could possibly help in those cases